src/test/java/com/rockwell/basics/animals/LionTest.java

```diff
@@ -0,0 +1,41 @@
+package com.rockwell.basics.animals;
+
+import com.rockwell.basics.animals.mammal.Lion;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+
+public class LionTest {
+
+    Lion lion = new Lion();
```

Comment:
tests should independant entities. If our logic will grew such property will be modifed inside each test. Who knows in which state the lion object will be after the test is finished. We want our code to be "stateless". Pleasem check what does "stateless" term mean. It will benefit in the future where we will jump into microservices world.